Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/git_hub.py

```diff
@@ -139,7 +139,9 @@ def create(self, head, title, body=None, commits=None, base=None, draft=None):
                 if not value:
                     raise ValueError("Must define '{}' when creating pull-request".format(key))
 
-            user, _ = self.repository.credentials(required=True)
+            if ':' not in head:
+                user, _ = self.repository.credentials(required=True)
+                head = f'user:{head}'
```

Comment:
Not everything running `webkitpy` has f-string support yet, so this will need to be be `.format`. Also, shouldn't this be `'{}:{}'.format(user, head)`?